ReactAndroid/src/main/java/com/facebook/react/views/webview/ReactWebViewManager.java

```diff
@@ -277,6 +278,11 @@ public void setUserAgent(WebView view, @Nullable String userAgent) {
     }
   }
 
+  @ReactProp(name = "mediaPlaybackRequiresUserAction")
+  public void setMediaPlaybackRequiresUserAction(WebView view, boolean requires) {
+    view.getSettings().setMediaPlaybackRequiresUserGesture(requires);
```

Comment:
I don't think `getSettings` does return null. `setDomStorageEnabled` does pretty much the same thing and doesn't check for example